buzzmobile/plan/steering/steering.py

```diff
@@ -47,12 +47,12 @@ def steering_node():
 def steer(ros_world_model):
     # convert RosImage to cv2
     try:
-        world_frame = bridge.imgmsg_to_cv2(ros_world_model)
+        world_frame = bridge.imgmsg_to_cv2(ros_world_model, 'mono8')
     except CvBridgeError:
         rospy.loginfo('Error converting world_model to cv2')
 
     # pick tentacle
-    height, width = world_frame.shape
+    height, width, _ = world_frame.shape
```

Comment:
So I actually think this will break if we are not mocking world_model. Can you revert this change and instead call `np.squeeze()` on `world_frame`?

aka:

```python
try:
    world_frame = np.squeeze(bridge.imgmsg_to_cv2(ros_world_model, 'mono8'))
```

or something like it